.travis.yml

```diff
@@ -64,41 +68,41 @@ jobs:
       language: python
       env: [ TEST_SUITE=build, 'SPEC=mpich' ]
 # astyle (MakefilePackage)
-    - python: '3.7'
+    - python: '3.8'
       os: linux
       language: python
       env: [ TEST_SUITE=build, 'SPEC=astyle' ]
 # tut (WafPackage)
-    - python: '3.7'
+    - python: '3.8'
       os: linux
       language: python
       env: [ TEST_SUITE=build, 'SPEC=tut' ]
 # py-setuptools (PythonPackage)
-    - python: '3.7'
+    - python: '3.8'
       os: linux
       language: python
       env: [ TEST_SUITE=build, 'SPEC=py-setuptools' ]
 # perl-dbi (PerlPackage)
-#    - python: '2.7'
+#    - python: '3.8'
 #      os: linux
 #      language: python
 #      env: [ TEST_SUITE=build, 'SPEC=perl-dbi' ]
 # openjpeg (CMakePackage + external cmake)
-    - python: '3.7'
+    - python: '3.8'
       os: linux
       language: python
       env: [ TEST_SUITE=build, 'SPEC=openjpeg' ]
 # r-rcpp (RPackage + external R)
-    - python: '3.7'
+    - python: '3.8'
       os: linux
       language: python
       env: [ TEST_SUITE=build, 'SPEC=r-rcpp' ]
 # mpich (AutotoolsPackage)
-    - python: '3.7'
+    - python: '3.8'
       os: linux
       language: python
       env: [ TEST_SUITE=build, 'SPEC=mpich' ]
-    - python: '3.6'
+    - python: '3.8'
```

Comment:
Is this docker test actually run?

Reply:
it might be that we still use this to push `packages.spack.io` - have to migrate this over some time soonish

Reply:
As far as I understand the docker test is a CD step that is always run on push to `develop` and `master`:

https://github.com/spack/spack/blob/4626c28c2735a8b6243b39270e55b9cdfc26e0da/.travis.yml#L113-L114

In fact at every commit we currently build [many docker images on Dockerhub](https://cloud.docker.com/u/spack/repository/list) (each with a different base OS).

Reply:
If you check the [Build History](https://travis-ci.org/spack/spack/builds/604792180?utm_medium=notification&utm_source=github_status) on Travis you should see the docker step run last.

Reply:
Yes, but we are in the process of migrating those builds to automatic builds on Dockerhub itself. For now, we still need it.